Wikipedia/Code/WMFFindInPageKeyboardBar.m

```diff
@@ -110,9 +110,9 @@ - (void)updateLabelTextForCurrentMatchIndex:(NSInteger)index matchesCount:(NSUIn
     } else if (count > 0 && index == -1) {
         labelText = [NSString stringWithFormat:@"%lu", (unsigned long)count];
     } else if (count == 0) {
-        labelText = MWLocalizedString(@"find-in-page-no-matches", nil);
+        labelText = [NSString stringWithFormat:MWLocalizedString(@"find-in-page-number-matches", nil), 0, 0];
     } else {
-        labelText = [NSString stringWithFormat:@"%lu / %lu", (unsigned long)(index + 1), (unsigned long)count];
+        labelText = [NSString stringWithFormat:MWLocalizedString(@"find-in-page-number-matches", nil), (unsigned long)(index + 1), (unsigned long)count];
```

Comment:
Since the localization string needs to change to  ` $1 / $2 `, this also needs to be updated to replace `$1` and `$2` in the string. Another potential (but not fully necessary) enhancement would be to use `[NSNumberFormatter localizedStringFromNumber:@(index + 1) numberStyle:NSNumberFormatterDecimalStyle]` to format the numbers before inserting them into the string (this will ensure that thousands separators are added if necessary).